source/getting-started/installation-raspberry-pi.markdown

```diff
@@ -1,29 +1,151 @@
+```bash
+(homeassistant_venv) homeassistant@raspberrypi:/srv/homeassistant $ pip3 install homeassistant
+```
+
+Start Home Assistant for the first time. This will complete the installation, create the `.homeasssistant` configuration directory in the `/home/homeassistant` directory and install any basic dependencies.
+```bash
+(homeassistant_venv) $ hass
+```
+
+You can now reach your installation on your raspberry pi over the web interface on [http://ipaddress:8123](http://ipaddress:8123). 
+For instruction on how to configure Home Assistant continue on with [Configuring Home Assistant](/getting-started/configuration/).
+
+### {% linkable_title Raspberry Pi Hardware Specific Components %}
+
+Some components that are specific for the Raspberry Pi can require some further configuration outside of Home Assistant. All commands below are assumed to be executed with the `pi` account. For full documentation of these components refer to the [components](/components) page.
+
+### {% linkable_title Bluetooth Tracker %}
+The Bluetooth tracker will work on a Raspberry Pi 3 with the built-in Bluetooth module or with a USB Bluetooth device on any of the other Raspberry Pi's.   
+
+Install the following external dependencies.
+```bash
+$ sudo apt-get install bluetooth libbluetooth-dev
+```
+After this follow the [Bluetooth Tracker component](/components/device_tracker.bluetooth_tracker/) and [Device Tracker page](/components/device_tracker/) pages.
+
+
+### {% linkable_title Raspberry Pi Camera %}
+The Raspberry Pi Camera is a specific camera for the Raspberry Pi boards. For more information about the camera I suggest reading the [documentation](https://www.raspberrypi.org/documentation/usage/camera/) from the Raspberry Pi foundation.  
+
+To use the camera it needs to be enabled with the `raspi-config` utility.
+```bash
+$ sudo raspi-config
+```
+Select `Enable camera` choose `<Yes>` and hit `Enter`, then go to `Finish` and you'll be prompted to reboot.  
+After reboot add your `homeassistant` account to the `video` group.
+```bash
+$ sudo adduser homeassistant video
+```
+After this follow the [Raspberry Pi Camera component](/components/camera.rpi_camera/) page.
 
- - Install Home Assistant
- - Launch Home Assistant and serve the web interface on [http://localhost:8123](http://localhost:8123)
+### {% linkable_title Raspberry Pi GPIO %}
+Each of the following devices are connected to the GPIO pins on the Raspberry Pi.
+For more details about the GPIO layout, visit the Wikipedia [article](https://en.wikipedia.org/wiki/Raspberry_Pi#GPIO_connector) about the Raspberry Pi.  
+Since these are not normally used some extra permission will need to be added.
+In general the permission that is needed is granted by adding the `homeassistant` account to the `gpio` group.
 
-There is also a [video tutorial](https://www.youtube.com/watch?v=GjzOXkPb7XE) created by [brusc](https://github.com/brusc).
+
+### {% linkable_title Raspberry Pi Cover %}
```

Comment:
Only for GPIO, so you have this type of layout

- Bluetooth tracker
- Camera
- GPIO related
  - DHT
  - Cover
  - Switch